src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -54,8 +54,8 @@ const PaymentAgentWithdraw = (() => {
 
             const currency = Client.get('currency');
             const form_id  = `#${$(view_ids.form).find('form').attr('id')}`;
-            const min      = isCryptocurrency(currency) ? 0.002 : 10;
-            const max      = isCryptocurrency(currency) ? 5 : 2000;
+            const min      = isCryptocurrency(currency) && currency !== 'DAI' ? 0.002 : 10;
+            const max      = isCryptocurrency(currency) && currency !== 'DAI' ? 5 : 2000;
```

Comment:
can you think of a better way than this?

Reply:
I didn't want to update the isCryptocurrency() function since it is used in so many places throughout the application and DAI is a cryptocurrency, I could write a new function for setting the min and max that takes currency as input?

Reply:
`getDecimalPlaces()` makes exactly the same categorization needed here, but not a good idea as it's not explicitly related.

Probably trying at least to not repeating the same condition? :)
I'd also suggest to put these kind of constatnt values at the top of every module and hopefully later refactor all to a single config file.